README.md

```diff
@@ -37,4 +37,26 @@ Praharsh Patel </br>
     <li> Straight line drawing </li>
 </ul>
 
+## Version 0.2.0
+### Features
+<ul>
+    <li> Added a button to allow drawing a straight line</li>
+    <li> Added a button to allow the user to select an image</li>
+    <li> Added ability to detect whether the file is an image type or not</li>
+</ul>
+
+### Bug Fixes
+<ul>
+    <li> <em>Image and tool bar are seprate, but previosly they were overlapping.</em> </li>
+    <li> <em>The background was defaulted to yellow but now is white.</em> </li>
+    <li> <em>The width of the images wasn't the correct size type, but now is.</em> </li>
+</ul>
+
+### Known Issues
+<ul>
+    <li> Image uploading only throws an error rather than promt the user to select a different file.</li>
```

Comment:
typo here can you fix :)

Reply:
Thanks for catching that, I changed promt to prompt.